docs/latest/examples/active-links.md

```diff
@@ -41,7 +41,7 @@ Twind. For Tailwind, use the syntax:
 ```tsx
 function Menu() {
   return (
-    <a href="/foo" class="aria-[current]:text-green-600">
+    <a href="/foo" class="aria-[current='page']:text-green-600">
```

Comment:
The selector here is already correct. Looks like it was already resolved by https://github.com/denoland/fresh/pull/2417 . Previously, we didn't have the distinction between twind and tailwindcss but that PR added that.

Reply:
Ah, ok then!

I was thinking I'd run into this before, but now that I think about it, I think it was the aria vs data distinction, not the value of the aria.